services/solr-names-updater/src/solr_names_updater/worker.py

```diff
@@ -66,7 +67,9 @@ def is_names_event_msg_type(msg: dict):
 
 def is_processable(msg: dict):
     """Determine if message is processable using message type of msg."""
-    if msg and is_names_event_msg_type(msg):
+    nr_num = msg.get('nrNum', None)
+    nr = RequestDAO.find_by_nr(nr_num)
+    if msg and is_names_event_msg_type(msg) and nr.entity_type_cd not in ('FR', 'GP'):
```

Comment:
Not saying you need to make this change, but there is a small chance (super small) the NR is gone, but it gives me a chance to show you the walrus (goo goo ga joob) ```:=```
```suggestion
    if msg and is_names_event_msg_type(msg) \
       and (nr := RequestDAO.find_by_nr(nr_num)) \
       and nr.entity_type_cd not in ('FR', 'GP'):
```

Reply:
done.